client-js/rtvi/events.ts

```diff
@@ -4,141 +4,152 @@
  * SPDX-License-Identifier: BSD-2-Clause
  */
 
-import { RTVIClientConfigOption } from ".";
-import { LLMFunctionCallData } from "./helpers/llm";
+import { Participant, TransportState } from "./common_types";
 import {
-  BotLLMTextData,
   BotLLMSearchResponseData,
+  BotLLMTextData,
   BotReadyData,
   BotTTSTextData,
+  LLMFunctionCallData,
   PipecatMetricsData,
   RTVIMessage,
-  StorageItemStoredData,
   TranscriptData,
 } from "./messages";
-import { Participant, TransportState } from "./transport";
 
 export enum RTVIEvent {
-  MessageError = "messageError",
-  Error = "error",
-
+  /** local connection state events */
   Connected = "connected",
   Disconnected = "disconnected",
   TransportStateChanged = "transportStateChanged",
 
-  Config = "config",
-  ConfigDescribe = "configDescribe",
-  ActionsAvailable = "actionsAvailable",
-
-  ParticipantConnected = "participantConnected",
-  ParticipantLeft = "participantLeft",
-  TrackStarted = "trackStarted",
-  TrackStopped = "trackStopped",
-  ScreenTrackStarted = "screenTrackStarted",
-  ScreenTrackStopped = "screenTrackStopped",
-  ScreenShareError = "screenShareError",
-
-  AvailableCamsUpdated = "availableCamsUpdated",
-  AvailableMicsUpdated = "availableMicsUpdated",
-  AvailableSpeakersUpdated = "availableSpeakersUpdated",
-  CamUpdated = "camUpdated",
-  MicUpdated = "micUpdated",
-  SpeakerUpdated = "speakerUpdated",
-
+  /** remote connection state events */
   BotConnected = "botConnected",
   BotReady = "botReady",
   BotDisconnected = "botDisconnected",
+  Error = "error",
+
+  /** server messaging */
+  ServerMessage = "serverMessage",
+  ServerResponse = "serverResponse",
+  MessageError = "messageError",
+
+  /** service events */
+  Metrics = "metrics",
+
+  // vad events
```

Comment:
nit: Perhaps streamline the comments to have the same format (`/** ... */` vs. `// ...`)

Reply:
ha. this was intentional but probably only made sense in my head. /** ... */ denote a larger class of event types and i was using // to denote sort of a sub-class of events under those.  So in this case, vad events are still service events, just related to the vad service. do you have thoughts on a better way to do this that's not obnoxious?

Reply:
I've definitely seen `/** ... */` denote larger sections. But they're usually spread across multiple lines to denote their bigness, otherwise there's no much visual distinction.

```
/**
 * SECTION
 */

// comment 1
// comment 2

/** 
 * NEXT SECTION
 */
```

Reply:
Ah, I didn't think of the "small" comments as subsections when reading it first. It makes sense now ☺️